src/styles/blocks/_section-footer.scss

```
@@ -0,0 +1,39 @@
+.section-footer {
+  display: block;
+  width: max-content;
+  margin: 0 auto;
+  position: relative;
+  @include font-properties("Raleway", 16px, 16px, 400);
+  color: $color-black;
+
+  transition: transform 0.3s;
```

You can create a variable for the duration
```suggestion
  transition: transform $duration;
```